python/ql/test/library-tests/frameworks/aiopg/test.py

```diff
@@ -0,0 +1,26 @@
+import aiopg
+
+# Only a cursor can execute sql.
+async def test_cursor():
+    # Create connection directly
+    conn = await aiopg.connect()
+    cur = await conn.cursor()
+    await cur.execute("sql")  # $ getSql="sql" constructedSql="sql"
+
+    # Create connection via pool
+    async with aiopg.create_pool() as pool:
+        # Create Cursor via Connection
+        async with pool.acquire() as conn:
+            cur = await conn.cursor()
+            await cur.execute("sql")  # $ getSql="sql" constructedSql="sql"
+
+        # Create Cursor directly
+        async with pool.cursor() as cur:
+            await cur.execute("sql")  # $ getSql="sql" constructedSql="sql"
+
```

Comment:
I looked at the implementation of [`_ContextManager.await`](https://github.com/aio-libs/aiopg/blob/c377089f617ebcfb36186be7381ef362e4a6d4f3/aiopg/utils.py#L78), and you get the same value from `x = await ctx` as you do from `async with ctx as x`, so we the code below _should_ also be valid, and I just thought it was nice to highlight that we handle it as well :blush: 

```suggestion

    # variants using as few `async with` as possible
    pool = await aiopg.create_pool()
    conn = pool.acquire()
    cur = await conn.cursor()
    await cur.execute("sql")  # $ getSql="sql" constructedSql="sql"

```